fixtures/run_as_non_root_false.yml

```diff
@@ -1,11 +1,14 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   creationTimestamp: null
   name: run_as_non_root_false
   namespace: fakeDeploymentRANR
 spec:
   strategy: {}
+  selector:
```

Comment:
Selectors are required since `apps/v1beta2` if we don't add the selector the Deployment Object worn't be used (since it's missing some required fields).

Reply:
Yes you're right, I missed that! This one lgtm.